test/data/manifests/rhel_84-x86_64-qcow2-customize.json

```diff
@@ -12,6 +12,11 @@
+      },
+      {
+        "name": "rt",
```

Comment:
This looks... unrelated. I think it's because of @achilleas-k's changes but I have no idea why this wasn't added before.

Reply:
Took me a while to figure out what was happening, thought a package was being added.

This is fine, we just added a new repo source for the rt kernel, but I neglected to generate all the existing test cases, so they weren't added.

Reply:
OK, thanks! :)

Reply:
What is the recommended way to regenerate the cases now? There are quite a few arches and distros now so it doesn't seem practical to regenerate the cases locally.

Reply:
There's a new script, `tools/test-case-generators/generate-all-test-cases`, that can generate all the test cases for all the arches inside VMs, but like you said, it's not very practical and takes hours.

Reply:
oh gosh haha. Let's see how this goes...